app/errors.js

```diff
@@ -16,5 +16,16 @@ exports.BAD_REQUEST = 'bad_request';
 exports.badRequest = message => internalError(message, exports.BAD_REQUEST);
 
 exports.PASSWORD_ENCRYPTION_ERROR = 'password_encryption_error.';
-exports.encryptionError = message =>
-  internalError(`Error trying to encrypt the password. ${message}`, exports.PASSWORD_ENCRYPTION_ERROR);
+exports.encryptionError = () =>
+  internalError('Error trying to encrypt the password.', exports.PASSWORD_ENCRYPTION_ERROR);
+
+exports.PASSWORD_DECRYPTION_ERROR = 'password_decryption_error.';
+exports.decryptionError = () =>
+  internalError('Error trying to decrypt the password.', exports.PASSWORD_DECRYPTION_ERROR);
+
+exports.SIGN_UP_ERROR = 'sign_up_error';
+exports.signUpError = message => internalError(message, exports.SIGN_UP_ERROR);
+
+exports.GENERATE_TOKEN_ERROR = 'generate_token_error.';
+exports.generateTokenError = () =>
+  internalError('Error trying to generate the token.', exports.GENERATE_TOKEN_ERROR);
```

Comment:
Move internal codes to top, and methods for bottom or on the contrary :+1: . This form is more organized.